src/ui.rs

```diff
@@ -641,6 +673,63 @@ impl<'a> UiFrame<'a> {
         color_token.pop(ui);
     }
 
+    pub fn draw_subdigital_logo(
+        &self,
+        tex_subdigital_logo: imgui::TextureId,
+        width_subdigital_logo: f32,
+        height_subdigital_logo: f32,
+    ) {
+        let ui = &self.imgui_ui;
+
+        let window_logical_size = ui.io().display_size;
+        let window_inner_width = window_logical_size[0] - 2.0 * MARGIN;
+        let window_inner_height = window_logical_size[1] - 2.0 * MARGIN;
+
+        let subdigital_logo_window_height_mult = height_subdigital_logo / width_subdigital_logo;
+
+        let subdigital_logo_window_height =
+            SUBDIGITAL_LOGO_WINDOW_WIDTH * subdigital_logo_window_height_mult;
+
+        let subdigital_logo_window_horizontal_position =
+            window_inner_width - SUBDIGITAL_LOGO_WINDOW_WIDTH;
+        let subdigital_logo_window_vertical_position =
+            window_inner_height - subdigital_logo_window_height * 0.8;
+
+        let color_token = ui.push_style_colors(&[
+            (imgui::StyleColor::WindowBg, self.colors.logo_window),
+            (imgui::StyleColor::Border, self.colors.logo_window),
+        ]);
+
+        imgui::Window::new(imgui::im_str!("Logo"))
+            .title_bar(false)
+            .movable(false)
+            .resizable(false)
+            .collapsible(false)
+            .size(
+                [
+                    SUBDIGITAL_LOGO_WINDOW_WIDTH + MARGIN,
+                    subdigital_logo_window_height,
+                ],
+                imgui::Condition::Always,
+            )
+            .position(
+                [
+                    subdigital_logo_window_horizontal_position,
+                    subdigital_logo_window_vertical_position,
+                ],
+                imgui::Condition::Always,
+            )
+            .build(ui, || {
+                imgui::Image::new(
+                    tex_subdigital_logo,
+                    [SUBDIGITAL_LOGO_WINDOW_WIDTH, subdigital_logo_window_height],
+                )
+                .build(ui);
```

Comment:
This image is almost invisible in the funky theme. Maybe we want to make a dark version of the logo, similarly to the logos in the about window?

Reply:
it's almost intentional. I'll check some other possibilities. maybe 50% grey would work for both.